Package.swift

```diff
@@ -61,7 +61,7 @@ switch ProcessInfo.processInfo.environment["SCUI_LIBRARY_TYPE"] {
 
 let package = Package(
     name: "swift-cross-ui",
-    platforms: [.macOS(.v10_15), .iOS(.v13), .tvOS(.v13), .macCatalyst(.v13)],
+    platforms: [.macOS(.v11), .iOS(.v13), .tvOS(.v13), .macCatalyst(.v13)],
```

Comment:
I feel like there was a reason for bumping to macOS 11 but I can't quite remember what it was. If it's at all possible we should use `if #available(...)` & co to maintain 10.15 compatibility.